controllers/quickstarts_test.go

```diff
@@ -115,3 +128,30 @@ func getActualQuickStarts(t *testing.T, cases []struct {
 	}
 	return allActualQuickStarts
 }
+
+func TestDeleteQuickStarts(t *testing.T) {
```

Comment:
Tests should be something like this https://github.com/red-hat-storage/odf-operator/blob/main/controllers/subscriptions_test.go#L29

and separate test and actual code into 2 commits

Reply:
Updated the tests, split into two commits

Reply:
Can you add and print label as well as shown in the above example

Reply:
done